Cabal/src/Distribution/PackageDescription/Check.hs

```diff
@@ -1576,10 +1576,10 @@ checkDevelopmentOnlyFlagsOptions fieldName ghcOptions =
 
     check has_WerrorWall $
       PackageDistInexcusable $
-           "'" ++ fieldName ++ ": -Wall -Werror' makes the package very easy to "
-        ++ "break with future GHC versions because new GHC versions often "
-        ++ "add new warnings. Use just '" ++ fieldName ++ ": -Wall' instead."
-        ++ extraExplanation
+           "'" ++ fieldName ++ ": -Werror with -W or -Wall makes the package "
+        ++ "very easy to break with future GHC versions because new GHC "
+        ++ "versions often add new warnings. Use just '" ++ fieldName
+        ++ ": -Wall' instead." ++ extraExplanation
```

Comment:
+1
Minor comment: this diff is larger than need be.  Only the first line changed, the other lines were rewrapped.  I'd prefer a half-full line over extra diff caused by wrapping.

Larger comment: Should the text "Use just ... -Wall instead" also be fixed?  I suggest to delete it simply.  The message to the user is not to use `-Werror`, what else the user chooses, is their own business.

Reply:
good suggestion, will fixup the commit